opt/scopeserver/dataserver/modules/gserver/GServer.py

```diff
@@ -581,56 +581,33 @@ def getMarkerGenes(self, request, context):
 
             def create_cluster_marker_metric(metric):
                 return loom.get_cluster_marker_metrics(
-                    clustering_id=request.clusteringID,
-                    cluster_id=request.clusterID,
-                    metric_accessor=metric["accessor"]
+                    clustering_id=request.clusteringID, cluster_id=request.clusterID, metric_accessor=metric["accessor"]
                 )
 
-            def to_pb(metric):
+            def protoize_cluster_marker_metric(metric):
                 return s_pb2.MarkerGenesMetric(
                     accessor=metric["accessor"],
                     name=metric["name"],
                     description=metric["description"],
-                    values=cluster_marker_metrics[metric["accessor"]]
+                    values=cluster_marker_metrics[metric["accessor"]],
```

Comment:
Why the trailing comma?

Reply:
I don't know seems that black is adding it...

Reply:
https://test-black.readthedocs.io/en/style-guide/style_guide/trailing_commas.html